tests/uptane_test.cc

```diff
@@ -153,9 +153,9 @@ TEST(uptane, sign) {
 }
 
 /*
- * \verify{\tst{153}} Check that aktualizr creates provisioning files if they
```

Comment:
I have clang-format version 3.8.1-18ubuntu1 (tags/RELEASE_381/final) and either way I format block comments (with or without the space after the first line), my clang-format-3.8 doesn't mess with it. I'm confused why yours does, but it's not worth losing sleep over.